schemas/network.yml

```diff
@@ -29,23 +19,15 @@
       example: 192.1.1.2
 
     # Metrics
-    - name: inbound.bytes
-      type: long
-      description: >
-        Network inbound bytes.
-      example: 184
-    - name: inbound.packets
-      type: long
-      description: >
-        Network inbound packets.
-      example: 12
-    - name: outbound.bytes
+    - name: transfer.bytes
       type: long
       description: >
-        Network outbound bytes.
+        Network bytes transferred. Typically used with netflow, sflow, or 
+        cloud provider flow logs
```

Comment:
We often use the first line as a quick description and then more details below. Can you reformat this as below and add a dot at the end?
```
Network bytes transferred. 

Typically used with netflow, sflow, or cloud provider flow logs.
```

Same applies to the description below.